### PR TITLE
Update Connect Request screen design

### LINKED ISSUE #5623: Implement "Connect per site" UI
As per https://github.com/MetaMask/Design/issues/36#issuecomment-432747776, we're ready to update the 1102 popup UI with final designs

image

### PR DESCRIPTION
**Pull request by whymarrh**
Closes #5623

This PR updates the design for the Connect Request screen with the latest design. (cc @cjeria)

**Screnshots**

### CODE REVIEW

**Comment by whymarrh** [comment]
A few notes:

- @bitpshr we can drop b63a6c9 before merging if you think that should be done differently—I just needed to pass through the title and image for the design but I'm open to other approaches
- @cjeria where did we imagine that "Learn more" link pointing?
- @alextsg @danjm if we want I can separate 4db45a1 into its own PR against `develop`
- If we're cool with the copy here, I will remove `providerAPIRequest ` and `reviewProviderRequest` from the other locale files

**Comment by whymarrh** [comment]
Also @cjeria what would we like to display in place of an image when a site doesn't have on available?

**Review by danjm — APPROVED** [review]

**Review by bitpshr — APPROVED** [review]

**Comment by bitpshr** [comment]
This looks great @whymarrh. I believe the two beta test failures are real (just a matter of adjusting targeted button text.)

**Comment by cjeria** [comment]
@whymarrh Here's what I suggest doing cc @omnat @bdresser 

image

**Comment by cjeria** [comment]
@whymarrh some feedback

- Identicons are too far apart. Should be 36px on either side of the green check icon
- Font sizes look off - The descriptor text "This site is requesting access to.." font size should be at 12px, line at 18px.
- Missing the checkbox "Keep my connected to this site" is missing.

**Comment by whymarrh** [comment]
> Missing the checkbox "Keep my connected to this site" is missing.

I think that's being left out of the current implementation (@bitpshr?) so I left it out of the design.

I'll update the other bits of the design, thanks!

**Comment by whymarrh** [comment]
For the default state you've shown above, is that the user's identicon we're showing? If not, what address is that the identicon for?

**Comment by cjeria** [comment]
@whymarrh It would be the identicon of the dapp/contract the user is interacting with. It's my understanding that we generate identicons for each address that we send ETH to. Is it not possible to load the identicon of the public address I'm sending eth to in this screen?

File: app/scripts/contentscript.js [hunk 1]
```diff
@@ -285,3 +287,33 @@ function redirectToPhishingWarning () {
...
+
+  // Use the site's favicon if it exists
+  const shortcutIcon = document.querySelector('head > link[rel="shortcut icon"]')
+  if (shortcutIcon) {
+    return shortcutIcon.href
+  }
+
+  // Search through available icons in no particular order
+  const icons = Array.from(document.querySelectorAll('head > link[rel="icon"]'))
+  for (let i = 0; i < icons.length; i++) {
```
Comment (danjm) [inline app/scripts/contentscript.js]: Any reason for a `for` loop instead of `.find()`?

**Review by danjm — APPROVED** [review]

File: app/scripts/contentscript.js (hunk 1 above)
Comment (whymarrh) [reply app/scripts/contentscript.js]: Nope, I can rewrite it

**Comment by cjeria** [comment]
@bitpshr Makes sense! @whymarrh here's an update with solid grey bg-color: #777A87;

image

**Pull request merged**